<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoleculePropertiesDialog</class>
 <widget class="QDialog" name="MoleculePropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Molecule Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="formulaLabel">
        <property name="text">
         <string>Formula:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="atomCountLabel">
        <property name="text">
         <string>Atom Count:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="bondCountLabel">
        <property name="text">
         <string>Bond Count:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="bondCountValue">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="molarMassLabel">
        <property name="text">
         <string>Molar Mass:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="molarMassValue">
        <property name="text">
         <string>123.3 g/mol</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="formulaValue">
        <property name="text">
         <string>C2H6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="nameValue">
        <property name="text">
         <string>Ethane</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="atomCountValue">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Line Notation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="formatLabel">
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="lineFormatComboBox">
        <item>
         <property name="text">
          <string>InChI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>InChIKey</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SMILES</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTextEdit" name="lineFormatValue">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MoleculePropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MoleculePropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
